main.py

```diff
@@ -16,22 +17,29 @@ def plot_history(history):
     ax.legend()
     save_plot(fig, 'history.png')
 
+
 def evaluate_model(model, test_data, verbose=2):
     _, test_acc = model.evaluate(test_data, verbose=verbose)
     print(f'Accuracy on test set after {config.MODEL_NUM_EPOCHS} epoch(s):'
-        + f' {test_acc*100:.2f}%')
+          + f' {test_acc * 100:.2f}%')
+
 
 def main():
     train_data, test_data = load_dataset()
     model = compile_model(num_classes=len(train_data.class_names))
-    history = model.fit(
-        train_data,
-        validation_data=test_data,
-        epochs=config.MODEL_NUM_EPOCHS,
-        batch_size=config.MODEL_BATCH_SIZE,
-        workers=config.MODEL_WORKERS,
-    )
-    plot_history(history)
+    if has_saved_weights():
+        load_status = model.load_weights(f'./{config.PICKLE_PATH}/ckpt')
```

Comment:
EXTREME nit but can we remove the relative path and use a join here to be consistent. can make a `load_weights(model, checkpoint_name)` function to cut down on the imports in main